yoti-sdk-impl/src/main/java/com/yoti/api/client/docs/session/create/check/SimpleRequestedLivenessCheckBuilder.java

```diff
@@ -0,0 +1,38 @@
+package com.yoti.api.client.docs.session.create.check;
+
+import static com.yoti.api.client.spi.remote.util.Validation.notNullOrEmpty;
+
+import com.yoti.api.client.docs.DocScanConstants;
+
+public class SimpleRequestedLivenessCheckBuilder extends RequestedLivenessCheckBuilder {
+
+    private String livenessType;
+    private int maxRetries = 1;
+
+    @Override
+    public RequestedLivenessCheckBuilder forZoomLiveness() {
+        this.livenessType = DocScanConstants.ZOOM;
+        return this;
```

Comment:
Don't think it makes too much difference, but should we change this to:
```suggestion
        return forLivenessType(DocScanConstants.ZOOM);
```

_(says suggestions can't be used on multi-line commits, not sure if this will appear weirdly)_